samples/bluetooth/hci_rpmsg/bt_mesh_overlay.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_BT_EXT_ADV=y
+
+# Disable unused Bluetooth features
+CONFIG_BT_CTLR_DUP_FILTER_LEN=0
```

Comment:
This configuration is missing in `mesh_provisioner` sample. I guess it should be added there to have the same configuration on nRF52 and nRF53 platforms.